test/general-tests/signature-argument/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+signature_len=6
+signature="1 2 3 4 5 6"
+
+# compile signatures contract
+npx hardhat starknet-compile contracts/signatures.cairo
+
+# deploy contract
+output=$(npx hardhat starknet-deploy --wait --starknet-network "$NETWORK" starknet-artifacts/contracts/signatures.cairo/)
+echo $output
+
+address=$(echo $output | sed -r "s/.*Contract address: (\w*).*/\1/")
+
+
+echo "Address: $address"
+
+# test call
+call_output=$(npx hardhat starknet-call \
+    --contract "signatures" --address "$address" \
+    --function "get_signature" --starknet-network "$NETWORK" \
+    --signature "$signature")
+
+echo $call_output
+
+if [[ "$call_output" != *"$signature_len $signature"* ]]; then
```

Comment:
Why do pattern matching with the inequality operator?

Reply:
[e042f33](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/109/commits/e042f332aa977ff629e807f644bfe02a461996c9)